doc/sphinx/proof-engine/proof-handling.rst

```diff
@@ -160,59 +171,38 @@ list of assertion commands is given in :ref:`Assertions`. The command
    the statement. In other words ``Proof using e`` is equivalent to
    ``Proof using Type + e`` for any declaration expression ``e``.
 
-   .. cmdv:: Proof using {+ @ident } with @tactic
-
-      Combines in a single line :cmd:`Proof with` and :cmd:`Proof using`.
-
-      .. seealso:: :ref:`tactics-implicit-automation`
-
-   .. cmdv:: Proof using All
-
-      Use all section variables.
-
-   .. cmdv:: Proof using {? Type }
-
-      Use only section variables occurring in the statement.
-
-   .. cmdv:: Proof using Type*
-
-      The ``*`` operator computes the forward transitive closure. E.g. if the
-      variable ``H`` has type ``p < 5`` then ``H`` is in ``p*`` since ``p`` occurs in the type
-      of ``H``. ``Type*`` is the forward transitive closure of the entire set of
-      section variables occurring in the statement.
-
-   .. cmdv:: Proof using -({+ @ident })
-
-      Use all section variables except the list of :token:`ident`.
-
-   .. cmdv:: Proof using @collection__1 + @collection__2
-
-      Use section variables from the union of both collections.
-      See :ref:`nameaset` to know how to form a named collection.
-
-   .. cmdv:: Proof using @collection__1 - @collection__2
+   :n:`- @section_subset_expr50`
+     Use all section variables except those specified by :n:`@section_subset_expr50`
 
-      Use section variables which are in the first collection but not in the
-      second one.
+   :n:`@section_subset_expr0 + @section_subset_expr0`
+     Use section variables from the union of both collections.
+     See :ref:`nameaset` to see how to form a named collection.
 
-   .. cmdv:: Proof using @collection - ({+ @ident })
+   :n:`@section_subset_expr0 - @section_subset_expr0`
+     Use section variables which are in the first collection but not in the
+     second one.
 
-      Use section variables which are in the first collection but not in the
-      list of :token:`ident`.
+   :n:`{? * }`
+     Use the transitive closure of the specified collection.
 
-   .. cmdv:: Proof using @collection *
+   :n:`Type`
+     Use only section variables occurring in the statement.  Specifying :n:`*`
+     uses the forward transitive closure of all the section variables occurring
+     in the statement. For example, if the variable ``H`` has type ``p < 5`` then
+     ``H`` is in ``p*`` since ``p`` occurs in the type of ``H``.
 
-      Use section variables in the forward transitive closure of the collection.
-      The ``*`` operator binds stronger than ``+`` and ``-``.
+   :n:`All`
+     Use all section variables.
 
+   .. seealso:: :ref:`tactics-implicit-automation`
 
 Proof using options
 ```````````````````
 
 The following options modify the behavior of ``Proof using``.
 
 
-.. opt:: Default Proof Using "@collection"
+.. opt:: Default Proof Using "@section_subset_expr"
    :name: Default Proof Using
 
    Use :n:`@collection` as the default ``Proof using`` value. E.g. ``Set Default
```

Comment:
Still one use of `@collection` here. If you remove it from the `nitpick_ignore` list, it should avoid any remaining occurrence.